tests/de/_common.yaml

```diff
@@ -0,0 +1,23 @@
+language: de
+areas:
+- name: Küche
+  id: kitchen
+- name: Wohnstube
```

Comment:
```suggestion
- name: Wohnzimmer
```

Doesn't make a difference here. IMO this is the more common word.

Reply:
it depends, in which region you are living 🙈 
But yes you're right, "Wohnzimmer" is the common

Reply:
I'd agree with that. Wohnzimmer is more common.